README.md

```diff
@@ -1,173 +1,198 @@
-# pybsn

```

Comment:
I was gonna ask what changed here. The list of commits solved the puzzle for me:
> convert to UNIX lineendings

Reply:
Oops sorry, yes, pls review by commit.